app/views/service_providers/_certificate_expiration_td.html.erb

```diff
@@ -1,14 +1,12 @@
-<% cert = app.certificate %>
-<% if cert.not_after < Time.zone.now %>
-  <td style='color: white; background: red'>
-<% elsif cert.not_after < ServiceProviderCertificate.warning_period %>
-  <td style='color: black; background: yellow'>
-<% else %>
-  <td>
+<%
+ # locals: app
+ cert = app.certificate
+%>
+
+<%= content_tag(:td, class: cert.expiration_css_class) do %>
```

Comment:
The new CSS classes have lower specificity than the inline styles, so the red background isn't being applied and the cell appears empty:

![image](https://user-images.githubusercontent.com/1779930/113047647-927d1380-916f-11eb-9ffa-47ac2beba7c1.png)

Because there's a background color being applied by `.usa-table td`:

https://github.com/uswds/uswds/blob/1908d1391bc59410624ca1934cc70b7404e8f443/src/stylesheets/core/placeholders/_table.scss#L88

We might need to do either...

- Apply the class to an inner element of the `td`
- Override the base `td` styles with greater specificity, like ...

```scss
.certificate-expired,
.usa-table td.certificate-expired {
  // ...
}
```

Unfortunately I don't see an option for the suggestion of my prior comment about using utility classes, so we may need the custom classes after all.

Reply:
Ooof, great catch. I went with the more specific styles in 119a9fed329bc328fd4d3f4c05529971e1586dbd